### ISSUE TITLE
Cityscapes label2photo evaluation issue

### ISSUE THREAD

**Issue by parvanitis15** [junyanz/pytorch-CycleGAN-and-pix2pix#974]
Hello!

I am trying to reproduce the results in Table 1 of the pix2pix paper by evaluating the results generated from pix2pix on the Cityscapes (label2photo) dataset using your scripts.

I resized all leftImg8bit images and gtFine images (incl. *_color.png, *_labels.png and *_instances.png, although I probably only need labels) of the original dataset to (256, 256). I also renamed the generated images to the original images' names from leftImg8bit/val folder.

Although the generated images seemed quite realistic, the evaluation results were bad and pretty far from the reported ones in the paper. Suspecting that something was wrong with the evaluation, I ran the evaluation on the original resized images (created as *_real_B.png in the results and renamed) to generate the ground truth evaluation results. Results showed that something is wrong with my evaluation since I am getting similar results to what I got from the pix2pix generated images (which are bad). 

I am trying to understand what I'm doing wrong here. Which images exactly do I need to resize in the original dataset in order to correctly run the script? As stated above, I resized all leftImg8bit images and gtFine images. Did I only need to resize the left8bitImg images?

A little help would be appreciated.

Thank you in advance

**Comment by junyanz** [comment]
@tinghuiz  See this [Q&A](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/docs/qa.md#for-labels2photo-cityscapes-evaluation-why-does-the-pretrained-fcn-8s-model-not-work-well-on-the-original-cityscapes-input-images-150) for more details.
